app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+import os
+import shlex
+import shutil
```

Comment:
let's don't use extra modules. the purpose of this task is to ensure that you know how to work with files without some unknown modules